templates/base/paginate.tmpl

```diff
@@ -1,28 +1,30 @@
-{{$paginationLink := .Page.GetParams}}
+{{$paginationParams := .Page.GetParams}}
+{{$paginationLink := $.Link}}
+{{if eq $paginationLink AppSubUrl}}{{$paginationLink = print $paginationLink "/"}}{{end}}
```

Comment:
What if `$paginationLink` already has a trailing slash? Is that possible?

Reply:
`{{if eq $paginationLink AppSubUrl}}{{$paginationLink = print $paginationLink "/"}}{{end}}`


AppSubUrl never has trailing slash.